### PR TITLE
book: make sure that the ads are actually readable

### PR DESCRIPTION
**Pull request by Kijewski**
We are using a free service, after all.

### CODE REVIEW

File: book/update-theme.py [hunk 1]
```diff
@@ -13,13 +13,33 @@
 INDEX_HBS_PORT = 443
 INDEX_HBS_PROTO = "GET"
 INDEX_HBS_PATH = "/repos/rust-lang/mdBook/contents/src/theme/index.hbs"
+INDEX_HBS_DATA = None
 INDEX_HBS_HEADERS = {
     "user-agent": "Update index.hbs for +https://github.com/rinja-rs/rinja",
 }
 
+ETHICALADS_THEME = r'''
+<style>
```
Comment (GuillaumeGomez) [inline book/update-theme.py:22]: Don't we have a style file or something? It's pretty bad for loading speed to keep the style in a `style` node.

File: book/update-theme.py [hunk 2]
```diff
@@ -13,13 +13,33 @@
...
+.ethical-sidebar {
+  margin-left: auto; }
```
Comment (GuillaumeGomez) [inline book/update-theme.py:24]: Also what happened with the indent here. XD

Comment (GuillaumeGomez) [reply book/update-theme.py:24]: Well, please fix it. :p

Comment (Kijewski) [reply book/update-theme.py]: Fixed. :)

File: book/update-theme.py (hunk 1 above)
Comment (Kijewski) [reply book/update-theme.py]: I append the content to a css file now during readthedocs' build process. More difficult to test now, but it should be sane.

File: book/ethicalads-theme.css [hunk 3]
```diff
@@ -0,0 +1 @@
+.ethical-sidebar{margin-left:auto}.ethical-sidebar .ea-content{background:var(--bg) !important;color:var(--fg) !important}.ethical-sidebar .ea-content a:is(:link, :visited){color:var(--fg) !important}.ethical-sidebar .ea-content a:is(:hover, :active, :focus){color:var(--sidebar-active) !important}.ethical-sidebar .ea-content strong{color:var(--sidebar-active) !important}.ethical-sidebar .ea-callout a:is(:link, :visited){color:var(--sidebar-non-existant) !important}.ethical-sidebar .ea-callout a:is(:hover, :active, :focus){color:var(--sidebar-fg) !important}
```
Comment (GuillaumeGomez) [inline book/ethicalads-theme.css:1]: I will close my eyes and try to forget about this. :rofl:

**Review by GuillaumeGomez — APPROVED** [review]

**Comment by GuillaumeGomez** [comment]
Thanks!

**Comment by Kijewski** [comment]
Hm, I did not not have the gray background in my local tests. It's quite obtrusive, but well, it's an ad after all?

**Pull request merged**